### ISSUE TITLE
Executing test cases from inside the editor

### ISSUE THREAD

**Issue by uuppi** [millennialmedia/intellibot#178]
It would be very useful if you could execute robot tests the same way as jUnit tests.
![robottest](https://cloud.githubusercontent.com/assets/5576504/17802603/d1e97bd6-65f9-11e6-9dee-1721422097b5.png)

**Comment by aleblanc39** [comment]
@uuppi I have a script that can can help you with it. 

https://github.com/aleblanc39/robot-test/blob/master/python-scripts/execute-robot-tests.py

I can give you more instructions if you feel like trying it.

**Comment by uuppi** [comment]
@aleblanc39 Thanks, but I already have a script that can execute test cases by name, and a macro that selects the test case's name and executes the script.

**Comment by aleblanc39** [comment]
@uuppi Then you should be able to invoke your script directly from PyCharm. Maybe not as advanced as JUnit, but still just a matter of right-clicking in the editor.

**Comment by uuppi** [comment]
@aleblanc39 Yes, I know. I'm using IntelliJ IDEA and by macro I meant a macro recorded inside the IDE. I can execute a test by moving the cursor to the line of a test case's name and pressing Ctrl+Shift+Alt+R. I just thought it would save me one extra action if I could simply just click a button in the editor ;)
